api/src/main/java/org/pmiops/workbench/cohortbuilder/util/CriteriaLookupUtil.java

```diff
@@ -27,10 +27,7 @@
 
 /**
  * This lookup utility extracts all criteria groups in the given search request and produces a
- * lookup map from group parameter to set of matching child concept ids. This class was implemented
- * to consolidate logic between the CB query builders and elastic search filters. The ElasticFilters
- * class will need to be refactored to use this lookup util going forward:
- * TODO:https://precisionmedicineinitiative.atlassian.net/browse/RW-2875
+ * lookup map from group parameter to set of matching child concept ids.
  */
 public final class CriteriaLookupUtil {
```

Comment:
This class was used exclusively by the Elastic implementation. Should we remove it as well? 
Also, in `CBCriteriaDao` we have 4 methods that are exclusive to  `CriteriaLookupUtil`
`findCriteriaAncestors`
`findCriteriaParentsByDomainAndTypeAndParentConceptIds`
`findCriteriaLeavesAndParentsByDomainAndTypeAndParentIds`
`findCriteriaLeavesAndParentsByPath`

Reply:
Thanks - missed that. Removed.